### PR TITLE
Add github templates

### PR DESCRIPTION
**Pull request by FlorentinD**
Helping users to write good issues

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+<!-- Running `python_code` helps to provide vital details -->
+
+Python Client version: X.Y.Z
```
Comment (adamnsch) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]: The client only has major and minor now `X.Y`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+GDS version: X.Y.Z
+Neo4j version: X.Y.Z
```
Comment (adamnsch) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: We might be interested in Python version too?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Thank you for your contribution to the Graph Data Science Client project.
+
+<!-- Please include a summary of the change, such as, which issue was fixed or feature was added. 
+If relevant, link to the corresponding issue.
+Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+Before submitting this PR, please make sure:
+- [ ] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library
```
Comment (adamnsch) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: All the details are outlined in `CONTRIBUTING.md` so we might want to refer to that?

**Review by adamnsch — APPROVED** [review]
Nice :) Me and @Mats-SX deliberately did not add these templates when setting up the repo. The reason was that the threshold for opening something should be as low as possible. I think that's still valid. But I definitely also see the point of these templates :)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (adamnsch) [reply .github/ISSUE_TEMPLATE/bug_report.md:16]: Also we might want to say "`graphdatascience` library version" to be clearer?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (FlorentinD) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: good point :+1:

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (FlorentinD) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: added a reference :+1:

**Comment by FlorentinD** [comment]
Good point, I added a sentence in the issue template that links to our graph-data-science repository

**Pull request merged**